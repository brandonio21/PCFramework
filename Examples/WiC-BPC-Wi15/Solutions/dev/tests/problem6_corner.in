138 305 330
23 3 240
238 72 102
90 90 90
383 363 600
