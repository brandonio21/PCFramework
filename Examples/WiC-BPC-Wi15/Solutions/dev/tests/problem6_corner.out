10 24 20
2 14 35
7 3 58
12 0 0
2 14 35
